CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # cuML 0.13.0 (Date TBD)
 
 ## New Features
+- PR #1655: Adds python bindings for homogeneity score
+- PR #1687: Adds python bindings for mutual info score
```

Comment:
Should only update for this PR here 1687

Reply:
I think this is because I removed a trailing space on this line `PR #1655: Adds python bindings for homogeneity score`. I can undo it if necessary.